internal/job/job.go

```diff
@@ -16,23 +16,27 @@ const (
 	StateFailed = "failed"
 )
 
+var StatesAll = []string{StatePassed, StateComplete, StateFailed, StateError, StateInProgress, StateQueued}
```

Comment:
For consistency (see _DoneStates_).

```suggestion
var AllStates = []string{StatePassed, StateComplete, StateFailed, StateError, StateInProgress, StateQueued}
```